pkgs/top-level/all-packages.nix

```diff
@@ -18262,6 +18262,8 @@ with pkgs;
 
   aeron = callPackage ../servers/aeron { };
 
+  aeron-cpp = callPackage ../servers/aeron-cpp { };
+
```

Comment:
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md
```suggestion
```

Reply:
Moved to pkgs/by-name as suggested